util.go

```diff
@@ -15,8 +15,14 @@ import (
 const aclName = "ratelimit"
 
 type appConfig struct {
-	HookService hookService
-	Lists       IPLists
+	Main struct {
```

Comment:
To differentiate a `main` section in a TOML config file from everything else. I'll probably toss more things in there as time goes on.

My toml file will look something like this:

```toml
[main]

LogFormat = "HaproxyHTTP"

[LogFormatOptions]

HostRequestHeader = 2
UserAgentRequestHeader = 1
TrueClientIPRequestHeader = 7

```